### ISSUE TITLE
"make stats" fails because it cannot find scc

### ISSUE THREAD

**Issue by cirego** [git-town/git-town#1386]
This is probably something simple that's missing from the developer setup step. But when I run `make stats`, I get the following error:

```
xargs: scc: No such file or directory
make: *** [Makefile:80: stats] Error 127
```

**Comment by cirego** [comment]
I fixed this by running:

`go get -u github.com/boyter/scc/`

**Comment by kevgo** [comment]
Good catch @cirego! I have shipped #1392 to add this to the developer documentation.
